config/core/configmaps/features.yaml

```diff
@@ -176,3 +176,14 @@ data:
     # 1. Enabled: enabling write access for persistent volumes
     # 2. Disabled: disabling write access for persistent volumes
     kubernetes.podspec-persistent-volume-write: "disabled"
+
+    # Controls if the queue proxy podInfo is enabled or not
+    # Should be enabled when using queue proxy Options (Extensions).
+    #
+    # 1. "enabled": enable podInfo volume
+    # 2. "disabled": disable podInfo volume
+    # 3. "allowed": disable podInfo volume by default.
+    #   However, clients may enable the behavior on an individual Service by attaching
+    #   the following metadata annotation: "features.knative.dev/mount-podinfo":"enabled".
+    # See: https://knative.dev/docs/serving/feature-flags/#kubernetes-podinfo
+    kubernetes.mount-podinfo: "disabled"
```

Comment:
```
queueproxy.mount-podinfo
```

Reply:
`kubernetes.*` implies we are 'enabling' some k8s feature.

This is actually tweaking the environment of the queue proxy - so I think that `queueproxy` is the appropriate prefix